### PR TITLE
Wann90 changes

### PR DESCRIPTION
**Pull request by JosePizarro3**
I want to improve a bit the parser to make the method `parse_system` callable.

This is because many advanced many-body methods (like DMFT) don't store anywhere the system geometry, except for from the Wannier90 output files themselves. This way I can call `Wannier90Parser:parse_system` in more advanced parser.

### CODE REVIEW

File: electronicparsers/wannier90/parser.py [hunk 1]
```diff
@@ -140,21 +150,60 @@ def __init__(self):
...
+            'sp3d-3': [-4, 3],
+            'sp3d-4': [-4, 4],
+            'sp3d-5': [-4, 5],
+            'sp3d2-1': [-5, 1],
+            'sp3d2-2': [-5, 2],
+            'sp3d2-3': [-5, 3],
+            'sp3d2-4': [-5, 4],
+            'sp3d2-5': [-5, 5],
+            'sp3d2-6': [-5, 6]
+        }
```
Comment (lauri-codes) [inline electronicparsers/wannier90/parser.py]: You have added this variable, but none of your new code uses it?

**Review by lauri-codes — COMMENTED** [review]
Done with the review here. Is the pipeline error expected?

**Comment by JosePizarro3** [comment]
> Done with the review here. Is the pipeline error expected?

Shouldn't be, as that metainfo is defined in nomad. @ndaelman-hu @ladinesa do you have any idea of why pylint is not passing here?

**Comment by ladinesa** [comment]
This is expected as the target nomad is v1.0.8

**Comment by JosePizarro3** [comment]
Should we update it now to a higher, e.g. 1.1.5? Or something is preventing us of doing it so?

And also, this is just about modifying the yaml in .github/workflows? I am curious.

**Comment by ladinesa** [comment]
Yes, but the package should be released which I think is not.

**Comment by JosePizarro3** [comment]
Ok, then this is ready for merging. I deleted the extra unused lines, so I merge directly.

**Pull request merged**